app/views/plots/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Plots Index Page</h1>
+
+<% @plots.each do |plot| %>
+  <div id="plot-<%= plot.id %>">
+    <p>Plot Number: <%= plot.number %></p>
+      <% plot.plot_plants.each do |plot_plant| %>
```

Comment:
You shouldn't be looping through the plot_plants. We want the plants from the plots.